### DISCUSSION TITLE
观看完毕“稍后再看”列表里的视频时自动将其从列表中删除

### DISCUSSION THREAD

**Discussion by QiuLiang-99** [the1812/Bilibili-Evolved#4693] [功能建议 / Ideas]
手机版b站的稍后再看现在可以区分视频是已经观看完还是没观看了，电脑版没办法看到。不然添加一个快捷键一键删除当前正在播放的视频？原版b站的稍后再看删除视频的时候非常卡，点一下按钮有时候要么没反应，要么要等好久才有反应

**Comment by Geeyun-JY3** [comment]
这不是有这功能吗：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E8%87%AA%E5%8A%A8%E7%A7%BB%E5%87%BA%E7%A8%8D%E5%90%8E%E5%86%8D%E7%9C%8B

**Comment by QiuLiang-99** [comment]
啊这...我用插件的搜索功能没有搜到所以以为没有做这个功能，原来是要在这里找

**Reply by Geeyun-JY3** [reply]
脚本内应该是能搜索到的：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/96252d79-64ed-4488-a6f2-464a37936f9f)
